src/pip/_internal/vcs/__init__.py

```diff
@@ -363,7 +388,8 @@ def obtain(self, dest):
                 self.name,
                 self.repo_name,
             )
-            prompt = ('(i)gnore, (w)ipe, (b)ackup ', ('i', 'w', 'b'))
+            # https://github.com/python/mypy/issues/1174
+            prompt = ('(i)gnore, (w)ipe, (b)ackup ', ('i', 'w', 'b'))  # type: ignore  # noqa: E501
```

Comment:
Again, please wrap the line instead of adding a `# noqa`:

```suggestion
            prompt = (
                '(i)gnore, (w)ipe, (b)ackup ',
                ('i', 'w', 'b'),
            )  # type: ignore
```